### ISSUE TITLE
Error-alerts on i(Pad)OS: Add specifics

### ISSUE THREAD

**Issue by Thomas-U** [brentsimmons/NetNewsWire#1442]
After a manual refresh I just saw this alert:
![See articles in  “All Unread”](https://user-images.githubusercontent.com/171936/71180852-545a2a00-2273-11ea-9d03-7f8e7845965c.jpeg)
If feasible It would be great to get info which feed caused this alert.

**Comment by brentsimmons** [comment]
Do we actually want to do an error for a timeout on a feed with a local account? The Mac app doesn’t. We can assume that, in general, the feed will just get read the next time, and this error isn’t worth bringing to the attention of the user.

Another reason: say the network is bad, and like 200 feeds time out. We don’t want 200 alerts.

**Comment by Thomas-U** [comment]
No,sorry, I only meant that if I get an error I would also like to get a clue where it stems from.

**Comment by Thomas-U** [comment]
Should I close this?

**Comment by brentsimmons** [comment]
Let’s leave it open.
